### ISSUE TITLE
[Support]: frigate container doesn´t start

### ISSUE THREAD

**Issue by samsix** [blakeblackshear/frigate#3399]
### Describe the problem you are having

Hi,

today I wanted to restart my frigate docker container after I´ve done some config changes. Now the container doesn´t start even if I remove the changes.
I reduced my config to minimal and still the container doesn´t start. I also removed the container and downloaded it again, but still no luck. 
I can´t connect and the container stopps directly after starting.

### Version

0.10.01

### Frigate config file

```yaml
# yaml-language-server: $schema=http://frigate_host:5000/api/config/schema
mqtt:
  host: IP
  port: 1888
detectors:
  cpu1:
    type: cpu
    num_threads: 4
cameras:
  Haustur: # <------ Name the camera
    ffmpeg:
      inputs:
        - path: http://IP/flv?port=1935&app=bcs&stream=channel0_sub.bcs&user=***&password=***# <----- Update for your camera
          roles:
            - detect
            - rtmp
        - path: http://IP/flv?port=1935&app=bcs&stream=channel0_main.bcs&user=***&password=***
          roles:
            - record
    rtmp:
      enabled: False # <-- RTMP should be disabled if your stream is not H264
    detect:
      width: 640 # <---- update for your camera's resolution
      height: 480 # <---- update for your camera's resolution
      fps: 4
    objects:
      track:
        - person
        - cat
        - dog
record:
  enabled: True
  events:
    retain:
      default: 10
      mode: active_objects
```

### Relevant log output

```shell
text  error  warn  system  array  login  

[2022-06-29 12:44:29] frigate.app                    INFO    : Starting Frigate (0.10.1-83481af)
Starting migrations
[2022-06-29 12:44:29] peewee_migrate                 INFO    : Starting migrations
There is nothing to migrate
[2022-06-29 12:44:29] peewee_migrate                 INFO    : There is nothing to migrate
[2022-06-29 12:44:29] detector.cpu1                  INFO    : Starting detection process: 224
[2022-06-29 12:44:29] frigate.app                    INFO    : Output process started: 226
[2022-06-29 12:44:29] frigate.app                    INFO    : Camera processor started for Haustur: 228
[2022-06-29 12:44:29] ws4py                          INFO    : Using epoll
[2022-06-29 12:44:29] frigate.edgetpu                WARNING : CPU detectors are not recommended and should only be used for testing or for trial purposes.
[2022-06-29 12:44:29] frigate.app                    INFO    : Capture process started for Haustur: 229
[2022-06-29 12:44:29] ws4py                          INFO    : Using epoll
[2022-06-29 12:44:29] frigate.mqtt                   INFO    : Turning off recordings for Haustur via mqtt
[2022-06-29 12:45:59] frigate.app                    INFO    : Starting Frigate (0.10.1-83481af)
Starting migrations
[2022-06-29 12:45:59] peewee_migrate                 INFO    : Starting migrations
There is nothing to migrate
[2022-06-29 12:45:59] peewee_migrate                 INFO    : There is nothing to migrate
[2022-06-29 12:45:59] detector.cpu1                  INFO    : Starting detection process: 225
[2022-06-29 12:45:59] frigate.app                    INFO    : Output process started: 227
[2022-06-29 12:45:59] ws4py                          INFO    : Using epoll
[2022-06-29 12:45:59] frigate.edgetpu                WARNING : CPU detectors are not recommended and should only be used for testing or for trial purposes.
[2022-06-29 12:45:59] frigate.app                    INFO    : Camera processor started for Haustur: 231
[2022-06-29 12:45:59] frigate.app                    INFO    : Capture process started for Haustur: 233
[2022-06-29 12:45:59] ws4py                          INFO    : Using epoll
[2022-06-29 12:45:59] frigate.mqtt                   INFO    : Turning off recordings for Haustur via mqtt
[2022-06-29 12:46:44] frigate.app                    INFO    : Starting Frigate (0.10.1-83481af)
Starting migrations
[2022-06-29 12:46:44] peewee_migrate                 INFO    : Starting migrations
There is nothing to migrate
[2022-06-29 12:46:44] peewee_migrate                 INFO    : There is nothing to migrate
[2022-06-29 12:46:44] detector.cpu1                  INFO    : Starting detection process: 225
[2022-06-29 12:46:44] frigate.app                    INFO    : Output process started: 227
[2022-06-29 12:46:44] frigate.edgetpu                WARNING : CPU detectors are not recommended and should only be used for testing or for trial purposes.
[2022-06-29 12:46:44] ws4py                          INFO    : Using epoll
[2022-06-29 12:46:44] frigate.app                    INFO    : Camera processor started for Haustur: 231
[2022-06-29 12:46:44] frigate.app                    INFO    : Capture process started for Haustur: 233
[2022-06-29 12:46:44] ws4py                          INFO    : Using epoll
[2022-06-29 12:46:44] frigate.mqtt                   INFO    : Turning off recordings for Haustur via mqtt
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[cont-finish.d] executing container finish scripts...
[cont-finish.d] done.
[s6-finish] waiting for services.
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[cont-finish.d] executing container finish scripts...
[cont-finish.d] done.
[s6-finish] waiting for services.
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[cont-finish.d] executing container finish scripts...
[cont-finish.d] done.
[s6-finish] waiting for services.
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.
```

### FFprobe output from your camera

```shell
not avaiable
```

### Operating system

UNRAID

### Install method

Docker Compose

### Coral version

CPU (no coral)

### Network connection

Wired

### Camera make and model

Reolink 520

**Comment by NickM-27** [comment]
Is your MQTT server accessible? I notice there's no password and the port isn't 1883 which is atypical. 

Also you disable rtmp but have it as a role which is wrong (not sure it'd cause an issue but could)

**Comment by samsix** [comment]
Yes that my MQTT-Testserver without credentials and on an non standard port. It´s accessable and in the logs I can see a short sign of life from the frigate server.

Client [frigate] connection closed: closed
Client [frigate] publishOnSubscribe send all known states
Client [frigate] subscribes on "frigate/#" with regex /^mqtt\.0\.frigate(\..*)?/
Client [frigate] subscribes on "frigate/#" with regex /^frigate(\..*)?/

I mow removed the role rtmp, but nothing changes.

**Comment by blakeblackshear** [comment]
There isn't any sign of an issue in the frigate logs. Do you have any sort of log output in the kernel logs about something killing processes? You could also try deleting the container and letting compose create a new one.

**Comment by samsix** [comment]
I just checked the syslog from my unraid server. I just getting the "normal" messages for the network change caused by starting and stopping the container. No hints why the container is stopping.

docker0: port 7(veth06c320a) entered blocking state
docker0: port 7(veth06c320a) entered forwarding state
docker0: port 7(veth06c320a) entered disabled state

I removed the container and images with the unraid WebUI and installed it again from the community apps.
Is there a more clean/advanced why to reinstall the container ?
